django/bossspatialdb/views.py

```diff
@@ -79,13 +79,20 @@ def get(self, request, collection, experiment, channel, resolution, x_range, y_r
         else:
             iso = False
 
```

Comment:
Assign default value to access_mode here. If neither no-cache nor access_mode is in the request, you'll get an exception due to an undefined variable.

Reply:
Good catch, Didn't test making an API call without either of the params.